src/Map/src/DependencyInjection/UXMapExtension.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\DependencyInjection;
+
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ChildDefinition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\UX\Map\Bridge as MapBridge;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+class UXMapExtension extends Extension implements ConfigurationInterface, PrependExtensionInterface
+{
+    /**
+     * @var array<string, array{ package: class-string<MapBridge\AbstractPackage> }>
+     */
+    private static array $bridges = [
+        'google' => [
+            'package' => MapBridge\Google\Package::class,
+        ],
+        'leaflet' => [
+            'package' => MapBridge\Leaflet\Package::class,
+        ],
```

Comment:
Let's remove 'package' here, so this becomes a map<name, Package class>, wdyt ?

Reply:
Yes you're right